README.md

```diff
@@ -0,0 +1,171 @@
+to be `SBT_TASK_LIMIT`. If `SBT_TASK_LIMIT` is not set, then (by default) a value of 4 is chosen.
+
+Used by `CakePlatformPlugin`, `CakeStandardsPlugin` and `CakeBuildPlugin`.
+
+## `SBT_VOLATILE_TARGET`
+
+When defined, sets the base directory for SBT's `target` setting. This is typically used to relocate the SBT build
+`target` directory and the JVM's temporary directory (i.e. the `java.io.tmpdir` property). For example, SBT builds can
+be speeded up by ensuring that `target` is located on a RAM disk in continuous integration (e.g. Jenkins).
+
+Used by `CakePlatformPlugin`, `CakeStandardsPlugin` and `CakeBuildPlugin`.
+
+# Build Plugin Functionality
+
+`CakeBuildInfoPlugin`, `CakeDockerPlugin`, `CakeJavaAppPlugin` and `CakePlatformPlugin` are top level plugins. By
+enabling these plugins in your project, you enable their respective SBT functionality.
+
+Settings within these plugins can be modified or overridden within local project SBT files (see below for useful
+examples).
+
+## `CakeBuildInfoPlugin`: Standard Build Information for Runtime Code
+
+Top level plugin.
+
+Assumes that `git` is installed. Used to configure and load the `BuildInfo` object at runtime. This object
+contains information regarding the build such as the git commit hash, etc.
+
+### Plugin Configuration
+
+Configuration of this plugin should be avoided in local project SBT build files.
+
+### SBT Tasks
+
+No special tasks are enabled for this plugin.
+
+## `CakeBuildPlugin`: Standard Build and Library Dependencies
+
+No requirements.
+
+Enabling this plugin in a project provides:
+* standard build and compiler settings
+* version setting using git via the [SBT Dynamic Versioning plugin](https://github.com/dwijnand/sbt-dynver).
+
+All projects should enable this plugin (either directly or indirectly).
+
+### Plugin Configuration
+
+See the plugin's source code to understand the range of build settings that may be modified.
+
+### SBT Tasks
+
+No special tasks are enabled for this plugin.
+
+## `CakeDockerPlugin`: Docker Native Packager
+
+Top level plugin.
+
+Enabling this plugin in a project configures how Docker containers may be built. See
+[Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html) for more information. This plugin is
+typically used alongside the `CakeJavaAppPlugin`. 
+
+### Plugin Configuration
+
+Docker container version number may be specified at the SBT command line by setting the system property `tag`.
+
+The following configuration settings can be modified in projects that enable this plugin:
+* `dockerRepository` (`None` by default) - used to define where the container will be published
+
+### SBT Tasks
+
+The expected suite of SBT Native Packager Docker tasks are enabled (e.g. `docker:stage`, `docker:publishLocal` and
+`docker:publish`).
+
+## `CakeJavaAppPlugin`: Java Application Server Packaging
+
+Top level plugin.
```

Comment:
I don't know what "top level plugin" is supposed to mean.

You do not enable this plugin directly, like many of the other plugins it auto-loads when you enable its dependency plugin, in this case `DockerPlugin`. There is no harm in loading, e.g. `CakeDockerPlugin`, manually, but it is extra characters for no good reason.